src/scene/scene.js

```diff
@@ -176,6 +183,95 @@ class Scene extends EventHandler {
         this.off();
     }
 
+    get clusteredLightingEnabled() {
+        return this._clusteredLightingEnabled;
+    }
+
+    set clusteredLightingEnabled(value) {
+
+        if (this._clusteredLightingEnabled && !value) {
+            console.error("Turning off enabled clustered lighting is not currently supported");
+            return;
+        }
+
+        this._clusteredLightingEnabled = value;
+    }
+
+    get lightCells() {
+        return this.worldClustersParams.cells;
+    }
+
+    set lightCells(value) {
+        this.worldClustersParams.cells.copy(value);
+    }
+
+    get maxLights() {
+        return this.worldClustersParams.maxLights;
+    }
+
+    set maxLights(value) {
+        this.worldClustersParams.maxLights = value;
+    }
```

Comment:
I wonder whether `maxLightsPerCell` might be better here, because otherwise, devs will probably assume it's per scene (since it's a property on the `Scene`). Also, is the valid range 0 to 255? If so, maybe clamp it?

Reply:
but it's similar to other properties, for example `shadowsEnabled` - this only affects clustered lights (omni and spot), but directional lights, and also non-clustered lights are not affected.

That's why I left the 'clustered' part of the name there. At some point the 'clustered' part won't be needed if we deprecate previous lighting system, but this will still work only for omni and spot (local lights) and not directional.

Maybe we use the `local` prefix? Not that it's very clear name .. it's just what engine uses internally. Or `nonDirectional`. Or maybe something related to the fact they are positioned (not just a direction)?

Reply:
That's true. I'm not too keen on adding `local` to any property names. @slimbuck + @Maksims - do you two have a preference here? Would be cool to get your thoughts.

Reply:
Originally I was thinking about simply exposing an instance of WorldClustersParams on the scene, perhaps as `scene.Lighting` or `scene.clusteredLighting`
https://github.com/playcanvas/engine/blob/ee00eca51c048fa91fad4aeaf1c9f1f5a36e599f/src/scene/lighting/world-clusters-params.js 

so then the properties would be
`scene.lighting.shadowEnabled = ..`

but then decided against it as the rest of the Scene properties are exposed directly .. even thought I'd probably prefer those to be grouped as well under `skybox`, `ambient`, 'lightmap' or similar.

Reply:
I agree 'maxLights' on the scene would sound misleading.

'maxLightsPerCell' could work, though a name that is less implementation specific would be even better.

Reply:
We could expose the clustered lighting parameters as `scene.clusteredLightingParams.maxLights` etc and then down the line deprecate that name and expose it as `scene.lightingParams` instead.

Reply:
Ideally we avoid creating an API now we know we need to deprecate - I'd much rather create one we can keep. I think we can simply use scene.lightingParams .. and make it clear in the documentation that these affect omni and spot lights.